cilium/cmd/bpf_ct_list.go

```diff
@@ -51,12 +56,46 @@ func init() {
 	command.AddOutputOption(bpfCtListCmd)
 }
 
-func getMaps(eID string) []*ctmap.Map {
-	if eID == "global" {
+func parseArgs(args []string) (string, uint32, error) {
+	t := args[0]
```

Comment:
We should check for at least two args to avoid index errors.

Reply:
Good point. I'll fix.